packages/db/convex/discord.ts

```diff
@@ -140,7 +140,26 @@ export const receiveMessage = apiMutation({
       threadId,
     });
 
-    //TODO: handle message type USER_JOIN
+    // If it's a new thread (where the message and thread IDs match) in the
+    // #support-community channel, send an auto-reply.
+    if (
+      dbThread &&
+      message.id === dbThread.id &&
+      channel.name === "support-community"
```

Comment:
```suggestion
      channel.id === process.env.AUTO_REPLY_CHANNEL_ID
```
In case we update the name again. I've set the env var in prod already